SRP/Example1.md

```diff
@@ -0,0 +1,22 @@
+### Usage
+
+---
+#### Where
+
+https://github.com/microsoft/vscode/blob/main/src/vs/editor/common/core/editOperation.ts
+
+#### Why
+
+https://github.com/microsoft/vscode/blob/main/src/vs/editor/common/core/editOperation.ts
+
+------------------------
+
+### Violation
+
+#### Where
+
+https://github.com/angular/angular/blob/main/packages/router/src/router.ts#L371
```

Comment:
➕
In my point of view it would be nice to create separate classes for error handling, url tree, routing, maybe something more.